lmdeploy/disagg/backend/backend.py

```diff
@@ -0,0 +1,12 @@
+from lmdeploy.disagg.config import MigrationBackend
+
+
+MIGRATION_BACKENDS = {}
```

Comment:
I think we can use the `mmengine` registry instead of making a new one, like we did in `lmdeploy/model.py`.
@RunningLeon what's your opinion?